app/services/api/v1x1/catalog/service_plan_reset.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1x1
+    module Catalog
+      class ServicePlanReset < Api::V1x0::Catalog::ServicePlanReset
+        attr_reader :reimported_service_plan
```

Comment:
I was initially thinking this name could be shorter, but after thinking about it more the descriptive name really helps clarify what you are getting so I am more in favor of it now.